---
  Add the lv5:rctrl_switch option to xkeyboard-config for Debian 12.
  This isn't in a release until xkeyboard-config v2.36.

  Apply it (as root) with:

    cd /usr/share/X11/xkb/rules
    patch -p1 < /path/to/this/file.patch

  Based on:
  https://github.com/freedesktop/xkeyboard-config/commit/da0a43b97aba73a6d60bb20d739b381748a56bba

diff -u a/base b/base
--- a/base      2023-01-22 17:03:50.604229639 +0000
+++ b/base      2023-01-22 17:04:13.051789551 +0000
@@ -1264,6 +1264,7 @@
   lv3:lsgt_switch_latch              =       +level3(lsgt_switch_latch)
   lv5:lsgt_switch                 =       +level5(lsgt_switch)
   lv5:ralt_switch                 =       +level5(ralt_switch)
+  lv5:rctrl_switch                =       +level5(rctrl_switch)
   lv5:lsgt_switch_lock            =       +level5(lsgt_switch_lock)
   lv5:ralt_switch_lock            =       +level5(ralt_switch_lock)
   lv5:lwin_switch_lock            =       +level5(lwin_switch_lock)
diff -u a/base.xml b/base.xml
--- a/base.xml  2023-01-22 17:03:50.608229561 +0000
+++ b/base.xml  2023-01-22 17:04:13.051789551 +0000
@@ -7858,6 +7858,12 @@
       </option>
       <option>
         <configItem>
+          <name>lv5:rctrl_switch</name>
+          <description>Right Ctrl chooses 5th level</description>
+        </configItem>
+      </option>
+      <option>
+        <configItem>
           <name>lv5:lsgt_switch_lock</name>
           <description>The "&lt; &gt;" key chooses 5th level and acts as a one-time lock if pressed with another 5th level chooser</description>
         </configItem>
diff -u a/evdev b/evdev
--- a/evdev     2023-01-22 17:03:50.608229561 +0000
+++ b/evdev     2023-01-22 17:04:13.055789472 +0000
@@ -1151,6 +1151,7 @@
   lv3:lsgt_switch_latch              =       +level3(lsgt_switch_latch)
   lv5:lsgt_switch                 =       +level5(lsgt_switch)
   lv5:ralt_switch                 =       +level5(ralt_switch)
+  lv5:rctrl_switch                =       +level5(rctrl_switch)
   lv5:lsgt_switch_lock            =       +level5(lsgt_switch_lock)
   lv5:ralt_switch_lock            =       +level5(ralt_switch_lock)
   lv5:lwin_switch_lock            =       +level5(lwin_switch_lock)
